user/templates/templates.md

```diff
@@ -13,10 +13,13 @@ title: Templates
 
 In [Getting Started](/doc/getting-started/), we covered the distinction
 in Qubes OS between where you *install* your software and where you *run* your
-software. Your software is installed in [templates](/doc/glossary/#template).
-Each template shares its root filesystem (i.e., all of its programs and system
-files) with all the qubes based on it. [App qubes](/doc/glossary/#app-qube) are
-where you run your software and store your data.
+software. Software that you use in most everyday tasks, is installed within [templates](/doc/glossary/#template).
+
+When using Qubes OS, you will typically begin a task by opening an application in an [app qube](/doc/glossary/#app-qube). App qubes are built from a *template* qube (or more simply, just *a template*). As such, app qubes they _inherit_ all data from their template—from the template's OS or distro, to installed applications, to stored files—so its "[root filesystem](https://opensource.com/life/16/10/introduction-linux-filesystems)." App qubes do not, however, _persist_ their data back to the Template. 
```

Comment:
I think the phrasing here might be a bit confusing for the intended audience.

Reply:
I guess that calls into question who the intended audience is—just developers, native Linux users, or also people new to Linux, as well? My own feeling is that it's currently written to support only very technical users, to the exclusion of folks unfamiliar with phrasing like "file systems" or "root." 

I feel like more technical users can do a better job working backwards to make sense of things, than more  novice users; and I only edited the introduction to make for an easier entry experience.